Android/What is savedInstanceState bundle.md

```diff
@@ -0,0 +1,39 @@
+
+ 1. **Bundle savedInstanceState**  is a reference to the Bundle object that is passed to the `onCreate()` in an Activity. Activities have capability to restore themselves to a previous state using the stored in the Bundle.
```

Comment:
I think if we will use - `` for nulls this improve readability. `null` looks much better than null, do you agree ?